packages/config/src/parse.js

```diff
@@ -65,13 +65,16 @@ const validateTomlBlackslashes = function (logs, configString, featureFlags) {
     return
   }
 
-  const [invalidSequence] = result
+  const [, invalidTripleSequence, invalidSequence = invalidTripleSequence] = result
```

Comment:
Uses either `(...)` group from the regular expression (but not `(?:...)` groups).